src/utils/Asset.ts

```diff
@@ -0,0 +1,87 @@
+import { ethers, Signer } from 'ethers'
+
+import { NftFactory } from '../contracts/NFTFactory'
+// eslint-disable-next-line import/no-named-default
+import ERC20Template from '@oceanprotocol/contracts/artifacts/contracts/interfaces/IERC20Template.sol/IERC20Template.json'
+import fs from 'fs'
+// eslint-disable-next-line import/no-named-default
+import { default as addrs } from '@oceanprotocol/contracts/addresses/address.json'
+
+/**
+ * Get the artifacts address from the address.json file
+ * either from the env or from the ocean-contracts dir
+ * @returns data or null
+ */
+export function getOceanArtifactsAdresses(): any {
+  try {
+    if (process.env.ADDRESS_FILE) {
+      // eslint-disable-next-line security/detect-non-literal-fs-filename
+      const data = fs.readFileSync(process.env.ADDRESS_FILE, 'utf8')
```

Comment:
i would also add a default here if the address file is not present and remove the condition above 

```suggestion
    const data = JSON.parse(
    // eslint-disable-next-line security/detect-non-literal-fs-filename
    fs.readFileSync(
      process.env.ADDRESS_FILE ||
        `${homedir}/.ocean/ocean-contracts/artifacts/address.json`,
      'utf8'
    )
  )
```

Reply:
can't see anything wrong here, what are you exactly suggesting @bogdanfazakas ?
this is perfectly fine
```
export function getOceanArtifactsAdresses(): any {
  try {
    if (process.env.ADDRESS_FILE) {
      // eslint-disable-next-line security/detect-non-literal-fs-filename
      const data = fs.readFileSync(process.env.ADDRESS_FILE, 'utf8')
      return JSON.parse(data)
    }
    return addrs
  } catch (error) {
    return addrs
  }
}
```
The `process.env.ADDRESS_FILE` should be the file path
if not defined or wrongly defined it returns the defaults from address.json, you're like repeating the same thing twice

Reply:
yep makes sense then, we can close this then